### PR TITLE
Coding convention applied split 2-7-2 (of 2-14-2)

### PR DESCRIPTION
**Pull request by Quix0r**
This PR is the next of **14** upcoming splits of the first series (yes, a lot has been changed) which started with #3254 (origin: #3112 ). It includes coding-convention stuff like added spaces and curly braces. Also some `dbm::is_result()` may have been included.

The commit has been split out from a larger commit containing 192 (!) changed files.

More similar changes: (in order they are created)

- [x] part 2 #3254 
- [x] part 2-1-2 #3287
- [x] part 2-2-2 #3369
- [x] part 2-3-2 #3375
- [x] part 2-4-2 #3381
- [x] part 2-5-2 #3469 
- [x] part 2-6-2 #3473 
- [ ] part 2-7-2 (this)
- [ ] part 2-8-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-8-2
- [ ] part 2-9-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-9-2
- [ ] part 2-10-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-10-2
- [ ] part 2-11-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-11-2
- [ ] part 2-12-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-12-2
- [ ] part 2-13-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-13-2
- [ ] part 2-14-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-14-2
- [ ] part 2-2 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split2-2 (**final!**)
- [ ] part 3 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split3
- [ ] part 4 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split4
- [ ] part 5 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split5
- [ ] part 6 https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-split6
- [ ] / : https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-root
- [ ] utils/ : https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-utils
- [ ] mod/ https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-mod
- [ ] include 1: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-include-if
- [ ] include 2: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-include-foreach
- [ ] others: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-others
- [ ] else-if: https://github.com/Quix0r/friendica/tree/rewrites/coding-convention-else-if

Unrelated but comes next:
- [ ] chmod https://github.com/Quix0r/friendica/tree/fixes/chmod

### CODE REVIEW

File: include/dfrn.php [hunk 1]
```diff
@@ -424,11 +426,17 @@ public static function relocate($owner, $uid) {
 
 		$rp = q("SELECT `resource-id` , `scale`, type FROM `photo`
 				WHERE `profile` = 1 AND `uid` = %d ORDER BY scale;", $uid);
+
```
Comment (annando) [inline include/dfrn.php]: This killing here is bad. These would only be missing profile pictures. So it would be enough to just don't add the xml elements.

File: include/dfrn.php [hunk 2]
```diff
@@ -1229,9 +1240,10 @@ public static function deliver($owner,$contact,$atom, $dissolve = false) {
 
 
 			if ($dfrn_version >= 2.1) {
-				if (($contact['duplex'] && strlen($contact['pubkey']))
-						|| ($owner['page-flags'] == PAGE_COMMUNITY && strlen($contact['pubkey']))
-						|| ($contact['rel'] == CONTACT_IS_SHARING && strlen($contact['pubkey']))) {
+				if (($contact['duplex'] && strlen($contact['pubkey'])) {
```
Comment (annando) [inline include/dfrn.php]: Why is this ```[``` here? This would surely create some syntax error on your machine. I guess that you don't actually work on your own code on your server?

File: include/email.php [hunk 3]
```diff
@@ -217,6 +217,7 @@ function email_header_encode($in_str, $charset) {
 
 	if (! $need_to_convert)
 		return $in_str;
+	}
```
Comment (annando) [inline include/email.php]: This is a closing brace without an opening one.

File: include/identity.php [hunk 4]
```diff
@@ -663,6 +668,10 @@ function advanced_profile(App $a) {
 			$profile['age'] = array( t('Age:'), $age );
 		}
 
+		if ($age = age($a->profile['dob'], $a->profile['timezone'], '')) {
```
Comment (annando) [inline include/identity.php]: Shouldn't it be ```==``` instead of ```=```? But even then I don't really understand the meaning.

File: include/text.php [hunk 5]
```diff
@@ -2213,6 +2219,10 @@ function text_highlight($s, $lang) {
 		$lang = 'javascript';
 	}
 
+	if (! strpos('Text_Highlighter',get_include_path())) {
```
Comment (annando) [inline include/text.php]: Why is this added here?

File: mod/_well_known.php [hunk 6]
```diff
@@ -37,17 +37,18 @@ function wk_social_relay(App $a) {
...
+		$server_tags = get_config('system', 'relay_server_tags');
 		$tagitems = explode(",", $server_tags);
 
-		foreach($tagitems AS $tag) {
+		foreach ($tagitems AS $tag) {
 			$tags[trim($tag, "# ")] = trim($tag, "# ");
 		}
 
-		if (Config::get('system', 'relay_user_tags')) {
+		if (get_config('system', 'relay_user_tags')) {
```
Comment (annando) [inline mod/_well_known.php]: Here you reverted something, it seems.

File: mod/_well_known.php [hunk 7]
```diff
@@ -37,17 +37,18 @@ function wk_social_relay(App $a) {
 	$tags = array();
 
 	if ($scope == SR_SCOPE_TAGS) {
-		$server_tags = Config::get('system', 'relay_server_tags');
+
+		$server_tags = get_config('system', 'relay_server_tags');
```
Comment (annando) [inline mod/_well_known.php]: At several places in this file you reverted the use of ```Config::```

File: mod/poco.php [hunk 8]
```diff
@@ -37,8 +37,8 @@ function poco_init(App $a) {
 		echo json_encode($ret);
 		killme();
 	}
+
 	if ($a->argc > 1 && $a->argv[1] === '@global') {
-		// List of all profiles that this server recently had data from
```
Comment (annando) [inline mod/poco.php]: Why had you removed this comment?

File: include/text.php (hunk 5 above)
Comment (Quix0r) [inline include/text.php]: Maybe I oversaw it. Will be removed.

File: mod/_well_known.php (hunk 7 above)
Comment (Quix0r) [inline mod/_well_known.php]: Yeah, need to undo that again. :-)

File: mod/_well_known.php (hunk 6 above)
Comment (Quix0r) [inline mod/_well_known.php]: All right by accident.

File: mod/events.php [hunk 9]
```diff
@@ -419,6 +419,7 @@ function events_content(App $a) {
 		if (x($_REQUEST, 'location'))    {$orig_event['location']    = $_REQUEST['location'];}
 		if (x($_REQUEST, 'start'))       {$orig_event['start']       = $_REQUEST['start'];}
 		if (x($_REQUEST, 'finish'))      {$orig_event['finish']      = $_REQUEST['finish'];}
+		if (x($_REQUEST,'finish')) $orig_event['finish'] = $_REQUEST['finish'];
```
Comment (Quix0r) [inline mod/events.php]: This needs to be removed. It is double.

File: mod/poco.php (hunk 8 above)
Comment (Quix0r) [inline mod/poco.php]: Opps ...

File: include/identity.php (hunk 4 above)
Comment (tobiasd) [inline include/identity.php]: Against what version of the code are you rebasing? See #3573  for the current code at the point, it still includes this statement, but also a check for not setting a age from the NULL_DATE.

@annando age() calculates the age but honouring the potential timezone shift I think..

File: include/acl_selectors.php [hunk 10]
```diff
@@ -607,8 +605,6 @@ function acl_lookup(App $a, $out_type = 'json') {
 		);
 		echo json_encode($o);
 		killme();
-	} else {
```
Comment (annando) [inline include/acl_selectors.php]: I hadn't seen where this array had been initialized.

File: include/dfrn.php [hunk 11]
```diff
@@ -113,6 +112,7 @@ public static function feed($dfrn_id, $owner_nick, $last_update, $direction = 0,
 		);
 
 		if (! dbm::is_result($r)) {
+			logger(sprintf('No contact found for nickname=%d', $owner_nick), LOGGER_WARNING);
```
Comment (annando) [inline include/dfrn.php]: See here for a list: https://github.com/friendica/friendica/blob/develop/boot.php#L114-L118

File: include/dfrn.php (hunk 1 above)
Comment (annando) [inline include/dfrn.php]: It would be better to just don't export the profile pictures when that query had been empty. This is better than committing suicide.

Concerning ```dbm::is_result```: It will return ```true``` whenever a query was successful or contains at least one line. This change was needed for the transition towards the new database functions.

Concerning I personally don't like a too high abstraction layer. There will always be the need for some low level calls.

File: include/dfrn.php (hunk 2 above)
Comment (annando) [inline include/dfrn.php]: Yeah, you should.

File: include/identity.php (hunk 4 above)
Comment (Quix0r) [inline include/identity.php]: @tobiasd upstream/master

**Comment by Quix0r** [comment]
I'm currently rebasing my `develop` branch ...

File: include/dfrn.php (hunk 1 above)
Comment (Quix0r) [inline include/dfrn.php]: You never wrote EJB/JPA/JSF code. ;-)

File: include/email.php (hunk 3 above)
Comment (Quix0r) [inline include/email.php]: The next block here has spaces instead of tabs for indenting code.

File: include/identity.php (hunk 4 above)
Comment (Quix0r) [inline include/identity.php]: @annando within the if() block, `$age` is being assigned to `$profile['age']`.

File: include/acl_selectors.php (hunk 10 above)
Comment (Quix0r) [inline include/acl_selectors.php]: Should be now very much up. I marked it with a comment there.

File: include/identity.php (hunk 4 above)
Comment (tobiasd) [inline include/identity.php]: Why are you rebasing against master when the development branch is the one that gets active changes? For me that sounds like the origin of all your reverts in the code.

Comment (Quix0r) [inline include/identity.php]: Maybe once I did it. Now only against `develop`. Maybe I got confused because many other projects I have cloned use `master` for active development.

**Comment by Quix0r** [comment]
Fixed more parts. I accidently committed `vendor/` because I ran `composer install` before ... That whole directory is normally (means, by many other projects) ignored and then everyone runs that said command. Just to repeat it again what it means to have "build" files being committed.

File: include/identity.php (hunk 4 above)
Comment (tobiasd) [inline include/identity.php]: I really was only curious, but that sounds like a reason :-)

File: include/dfrn.php (hunk 1 above)
Comment (annando) [inline include/dfrn.php]: No. Just assign the default avatars or don't include the avatars in the created XML. It would be very bad if a minor issue like a not existing avatar picture would block the transmission of the relocate message that is saying "I'm moving from server A to server B".

File: include/auth_ejabberd.php [hunk 12]
```diff
@@ -320,7 +324,7 @@ private function writeLog($sMessage) {
 	 * @param string $sMessage The logfile message
 	 */
 	private function writeDebugLog($sMessage) {
-		if ($this->bDebug)
+		if ($this->bDebug) {
```
Comment (annando) [inline include/auth_ejabberd.php]: This bracket is never closed.

It would be really good if you could check your pull requests on your local installation for syntax errors.

Comment (Hypolite) [inline include/auth_ejabberd.php]: I don't mind running `php util\typo.php` each time @Quix0r fixes the last syntax error, it's quick and painless for me. If it bothers him, he'll start doing it on his own after a while. 🙂

File: include/plugin.php [hunk 13]
```diff
@@ -353,12 +359,17 @@ function get_theme_info($theme){
 		'unsupported' => false
 	);
 
-	if (file_exists("view/theme/$theme/experimental"))
+	if (file_exists("view/theme/$theme/experimental")) {
 		$info['experimental'] = true;
-	if (file_exists("view/theme/$theme/unsupported"))
+	}
+
+	if(file_exists("view/theme/$theme/unsupported")) {
```
Comment (Quix0r) [inline include/plugin.php]: Args, you cannot merge arrays with a `+`. You must use `array_merge()` for it.

Sure I will add it.

File: include/dfrn.php (hunk 1 above)
Comment (Quix0r) [inline include/dfrn.php]: Ah, got it now. 👍

**Comment by annando** [comment]
619 changed files look like something went wrong :-)

File: include/dfrn.php (hunk 1 above)
Comment (Quix0r) [inline include/dfrn.php]: I rewrote the way, that it also prevent 3 `E_NOTICE` if photos are missing.

File: include/identity.php (hunk 4 above)
Comment (Quix0r) [inline include/identity.php]: Should be removed by now?

File: include/auth_ejabberd.php (hunk 12 above)
Comment (Quix0r) [inline include/auth_ejabberd.php]: I have now included it in my `pre-commit` hook. Nice work for that. :-)

And we should eliminate `?>` all together as it is discouraged.

**Comment by Quix0r** [comment]
F**k ... :-(

**Comment by Quix0r** [comment]
Garr... :-( I think I relaunch this PR. Your work is not for nothing, I included all your feedback into my next commits. :-)

**Comment by Quix0r** [comment]
I think the other ongoing developments (commits) have maybe caused it because then I try to prevent merges where I can to reduce GIT history complexity.

**Sorry for closing this one (again ...)!** But this is disastrous for a PR of such type.

**Comment by Quix0r** [comment]
For the record, I now split into much smaller chunks, only 5-8 commits each. This should work better and faster for reviewing them.
